pkg/domain/entities/pods.go

```diff
@@ -115,23 +115,23 @@ type PodSpec struct {
 // PodCreateOptions provides all possible options for creating a pod and its infra container
 // swagger:model PodCreateOptions
 type PodCreateOptions struct {
-	CGroupParent       string
-	CreateCommand      []string
-	Hostname           string
-	Infra              bool
-	InfraImage         string
-	InfraName          string
-	InfraCommand       string
-	InfraConmonPidFile string
-	Labels             map[string]string
-	Name               string
-	Net                *NetOptions
-	Share              []string
-	Pid                string
-	Cpus               float64
-	CpusetCpus         string
-	Userns             specgen.Namespace
-	Volume             []string
+	CGroupParent       string            `json:"cgroup_parent,omitempty"`
```

Comment:
Can you add a comment to the PodCreateOptions struct that the json tags are meant to match the respective fields in the ContainerCreateOptions struct.

Reply:
done